### DISCUSSION TITLE
Using Serverless Drivers with Cloudflare Workers

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [prisma/prisma#22304] [Q&A]
### Question

Hi, 
I was reading through the [announcement post](
https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV) on Prisma Blog and was wondering if is it possible to use the new Serverless Drivers with Cloudflare Workers? 

I've set everything up, enabled the `driversAdapters` feature in my schema but I can't get it to work in Cloudflare Workers. 

When I try to run that in a Worker, I get the following error: `PrismaClient is unable to run in Cloudflare Workers. As an alternative, try Accelerate: https://pris.ly/d/accelerate.`

Is there any way to get it working without Accelerate?
Thanks!

### Information about Prisma Schema, Client Queries and Environment (optional)

Here's my code:

```ts
// src/client.ts

import { remember } from "@epic-web/remember"
import { Client } from "@planetscale/database"
import { PrismaPlanetScale } from "@prisma/adapter-planetscale"
import { Prisma, PrismaClient } from "@prisma/client"
import { config } from "dotenv"

// Setup dotenv
config()

const connection = new Client({ url: process.env.DATABASE_URL! })
const adapter = new PrismaPlanetScale(connection)
return new PrismaClient({ adapter })
```

```prisma
// prisma/schema.prisma

generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["driverAdapters"]
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

...
```

- OS: macOS
- Database: MySQL
- Node.js version: 20.9.0
- Prisma version: 5.7.0

**Comment by janpio** [comment]
Not yet, but we are working on it: https://github.com/prisma/prisma/issues/21394 + https://github.com/prisma/prisma/issues/19488

**Comment by TimKieu** [comment]
So sad, I got this error today adter long coding. It seems that Prisma will lock it client to run via proxy. So many dependencies comes.
I will port all my project to Kysely which can run anywhere CloudFlare Workers/Vercel/ nodejs..

**Reply by janpio** [reply]
We are working on it and in Early Access right now: https://github.com/prisma/prisma/issues/21394
If you don't have the time to wait for that, Kysely is indeed a valid and good option.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
